tests/components/mobile_app/test_http_api.py

```diff
@@ -66,13 +67,6 @@ async def test_registration_encryption(
     hass: HomeAssistant, hass_client: ClientSessionGenerator
 ) -> None:
     """Test that registrations happen."""
-    try:
-        from nacl.encoding import Base64Encoder
-        from nacl.secret import SecretBox
-    except (ImportError, OSError):
-        pytest.skip("libnacl/libsodium is not installed")
-        return
-
```

Comment:
I split this out of #119389 because I wasn't sure if it was desirable.
But IMO since `nacl` always works in the CI there is no need for these checks.

Reply:
Makes sense IMHO.